src/kakarot/instructions/memory_operations.cairo

```diff
@@ -83,4 +85,29 @@ namespace MemoryOperations {
         let ctx = ExecutionContext.increment_gas_used(ctx, GAS_COST_PC);
         return ctx;
     }
+
+    // @notice MSIZE operation
+    // @dev Get the value of memory size.
+    // @custom:since Frontier
+    // @custom:group Stack Memory Storage and Flow operations.
+    // @custom:stack_produced_elements 1
+    // @return Updated execution context.
+    func exec_msize{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr,
+        bitwise_ptr: BitwiseBuiltin*,
+    }(ctx: model.ExecutionContext*) -> model.ExecutionContext* {
+        alloc_locals;
+        %{ print("0x59 - MSIZE") %}
+        let msize = Helpers.to_uint256(ctx.memory.raw_len);
```

Comment:
```suggestion
        let len = Memory.len(ctx.memory);
        let msize = Helpers.to_uint256(len);
```

Reply:
raw len is the number of felts in the memory, as Uint256 is a struct composed by 2 felts raw_len = 2 * actual_len.

Reply:
Oh Ok! Thank You!